isisd/isis_tlvs.c

```diff
@@ -1268,7 +1286,7 @@ static void format_item_ext_subtlvs(struct isis_ext_subtlvs *exts,
 		struct isis_srv6_endx_sid_subtlv *adj;
 
 		if (json) {
-			struct json_object *arr_adj_json, *flags_json;
+			struct json_object *arr_adj_json, *srv6_end_sid_json;
```

Comment:
Can we change the name to `srv6_endx_sid_json` or `srv6_end_x_sid_json`?

`srv6_end_sid_json` could create confusion.

According to the RFC terminology,
SRv6 End SID => Prefix SID
SRv6 End.X SID => Adjacency SID

Here we are showing the Adjacency SIDs (SRv6 End.X SIDs).

Reply:
done. I've set `srv6_endx_sid_json`